### DISCUSSION TITLE
ERR to many redirects and validation issues

### DISCUSSION THREAD

**Discussion by artworkad** [spatie/laravel-data#566] [Q&A]
I am tinkering with laravel data validations and have following setup:

```
class CatalogSearchRequest extends Data
{
    public function __construct(
        #[In(['countries', 'languages'])]
        public string $table,
        #[Min(20)]
        public string $searchTerm
    ) {
    }
}
```

Controller:

```
class CatalogController extends Controller
{
    public function __invoke(CatalogSearchRequest $request)
    {
        ray($request);
        return [];
    }
}
```

When I provide a search term that is too short, I get `ERR to many redirects error` in the browser. When I provide a table that is not in the array, a CatalogSearchRequest will be created anyway and fallback to countries. Shouldn't a validation error be thrown? Am I missing something?

Thx!

**Comment by rubenvanassche** [comment]
Could you provide me a test showing this behavior? Then I can take a further look, this should normally work and is fully tested in the package so a bit strange that this case would fail.

**Reply by artworkad** [reply]
I reproduced this in a fresh laravel app with just laravel-data and one controller: https://github.com/artworkad/laravel-data-validation. Let me know if this is sufficient for you to look into the issue.

**Reply by rubenvanassche** [reply]
Still can't reproduce: tried this http://laravel-data-validation.test/search?searchTerm=bla&table=countries
